third_party/crypto/xmake.lua

```diff
@@ -0,0 +1,6 @@
+compartment("crypto")
+    add_deps("freestanding", "debug", "cxxrt")
+
```

Comment:
I’d rather this lives in the demo for now. It’s not a generic crypto compartment, it’s quite specific to this use case.

Reply:
fair enough -  I had ambitions to create a much more comprehensive wrapper along the same lines that MQTT wraps the  FreeRTOS module, and this was just the smallest subset needed to land something easily for you ;-).   I'll add a simple ReadMe that describes how it can be included / initialised and move the comp wrapped back into my demo.

I'll wait a few days to see if the upstream changes land in case we can pull that in at the same time